lib/sorbet-rails/tasks/rails_rbi.rake

```diff
@@ -19,7 +19,7 @@ namespace :rails_rbi do
     root_dir = args[:root_dir] || Rails.root
     file_path = Rails.root.join("sorbet", "rails-rbi", "routes.rbi")
     FileUtils.mkdir_p(File.dirname(file_path))
-    File.write(file_path, inspector.format(SorbetRails::RoutesRbiFormatter.new))
+    File.write(file_path, inspector.format(SorbetRails::RoutesRbiFormatter.new).join("\n"))
```

Comment:
Why is this needed? The whole content is generated in RoutesRbiFormatter.

Reply:
It was outputting a single line with a bunch of `\n` before, though I don't think that's true if we remove the buffer.